README.md

```diff
@@ -1,4 +1,23 @@
 # Documentation
 
 This repository exists for the purpose of planning and organizing documentation
-changes and milestones for the Laminas Project and related subproject.
+changes and milestones for the Laminas Project and related subproject, and to show how to setup and update the documentation locally.
+
+## Setup
+
+To prepare your local development environment to build the documentation locally, follow the steps below:
+
+1. Install [Python 3](https://www.python.org/downloads/) and [PIP](https://pip.pypa.io/en/stable/installing/)
+2. Install the [Python PyMdown extension](https://facelessuser.github.io/pymdown-extensions/installation/#requirements)
+3. [Install MkDocs](https://www.mkdocs.org/#installation) and all of its dependencies
```

Comment:
A package manager should be used for the installation of Python and this is already included on mkdocs.org.

```suggestion
1. [Install MkDocs](https://www.mkdocs.org/#installation) and all of its dependencies (The installation description for MkDocs also includes the installation of Python.)
2. Install the [Python PyMdown extension](https://facelessuser.github.io/pymdown-extensions/installation/#installation_1)
```